### PR TITLE
fix: clean up breakpoints, grid and add docs

### PR DESCRIPTION
**Pull request by adamraider**
- adds breakpoints docs page explaining how to create a media query and use the variables
- removes the redundant variable map $ray-grid-breakpoints in favor of $ray-layout-grid-breakpoints
- renames ray-grid to ray-page-container
- renames ray-grid__inner to ray-grid

### CODE REVIEW

File: docs/components/grid.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ title: Grid System
...
-| `.ray-grid__cell`                                            | Mandatory, for the layout grid cell (default width: 4 columns)                                                                                                                                |
+| `.ray-page-container`                                        | Typically used once on a page, defines maximum width and padding charactersitics for the page                                                                      |
+| `.ray-page-container--align-<POSITION>`                      | Optional, align the grid to the left or right side of the container<br>`POSITION`: left, right                                                                     |
+| `.ray-page-container--justify-<POSITION>`                    | Optional, align the cells of the grid (default: left, does not affect nested grids<br>`POSITION`: left, center, right                                              |
+| `.ray-grid`                                                  | Mandatory, for wrapping grid cells (_must_ be child of `.ray-grid`)                                                                                                |
```
Comment (i8ramin) [inline docs/components/grid.md]: > (_must_ be child of `.ray-grid`)

is this still true?

Comment (adamraider) [reply docs/components/grid.md]: no!

File: packages/core/src/global/material-grid/mdc-layout-grid-debug.scss [hunk 2]
```diff
@@ -38,10 +38,10 @@
   );
 }
 
-.#{$ray-class-prefix}grid--debug {
+.#{$ray-class-prefix}page-container--debug {
```
Comment (i8ramin) [inline packages/core/src/global/material-grid/mdc-layout-grid-debug.scss]: looks like i gotta update my chrome extension!

Comment (adamraider) [reply packages/core/src/global/material-grid/mdc-layout-grid-debug.scss]: heh

**Pull request merged**